evadb/binder/statement_binder.py

```diff
@@ -331,9 +337,17 @@ def _bind_func_expr(self, node: FunctionExpression):
                 )
                 # certain functions take additional inputs like yolo needs the model_name
                 # these arguments are passed by the user as part of metadata
-                node.function = lambda: function_class(
-                    **get_metadata_properties(function_obj)
-                )
+                # we also handle the special case of ChatGPT where we need to send the
+                # OpenAPI key as part of the parameter
+                # ToDO: this should be better handled
```

Comment:
Do you think have a CREATE call like the following is the right approach? 
```
CREATE FUNCTION myllm
TYPE llm
MODEL GPT4
OPENAI key;
```
If so, we can mention in the TODO comment.

Reply:
This should already work.  Will handle it.

Reply:
Done!